QgisModelBaker/libqgsprojectgen/generator/generator.py

```diff
@@ -77,7 +77,42 @@ def append_print_message(self, level, text):
         if message not in self.collected_print_messages:
             self.collected_print_messages.append(message)
 
+    def omitted_layers(self):
```

Comment:
Would it be possible to

 - move this to the connector
 - combine this with `IGNORED_TABLES` -- ideally, this would be made more specific for each provider (e.g. right now it's also listing gpkg specific tables on postgres)

Reply:
For consistency I would also like to use either `omitted` or `ignored` as term for this.

Reply:
I can include the `IGNORED_TABLES` into the omit tables on creating the layers. But the ignored tables are used as well in `get_tables_info_without_ignored_tables` and I'm not sure if the omitted tables should be there as well. Btw. `get_tables_info_without_ignored_tables` is never called. You know anything about it? Can it be removed?

Reply:
But anyway shouldn't we keep the `IGNORED_TABLES` to have the "lists" separated. Just append them in the `def omitted_layers()` to the list of tables to have only one check in creating the layers.

Reply:
I don't remember. Can you look at the `git blame` for `get_tables_info_without_ignored_tables`?
It's ok to keep the lists separated (global static, provider specific static, dynamic) and then use them at the same time. What do you think?

Reply:
Yes. That's what I prefer.

Reply:
I did it but do we really want this stuff in the connector?

Reply:
Hm.. I'll have to import the config from the generator to the dbconnector or I need to create a dbconnector config file containing the IGNORED_TABLES. These are defined together with the IGNORED_SCHEMAS etc. so I would like to not split them.

Shouldn't we just handle it in the generator like suggested in the beginning to keep the db_connector clean @m-kuhn ?

Reply:
I think it makes sense in here because of provider specific logic. In here we can add custom ignored tables on a provider (connector) basis in subclasses which we cannot easily do without `if connector_pg elif connector_gpkg ...` etc code in the generator. Ignored_schemas are a very good candidate for moving here too because they are very provider specific.
Does that make sense?

Reply:
Okay, moving everything is fine as well for me.